spec/lib/embed/viewer/file_spec.rb

```diff
@@ -51,6 +51,13 @@
       html = Capybara.string(file_viewer.to_html)
       # visible false because we display:none the container until we've loaded the CSS.
       expect(html).to have_css '.sul-embed-file-list', visible: false
+      expect(html).to have_css 'a[download]', visible: false, text: '12.35 kB'
+    end
+    it 'with no file size' do
```

Comment:
Could we make the description represents the test? For example, "Displays download instead of the file size for the files with no size"